AppAuth-JS/app/app.css

```diff
@@ -0,0 +1,11 @@
+html {
```

Comment:
add license header

Reply:
this is Google's lib fork. Seems Google has missed license, and I did not edit this file